### PR TITLE
[office-js-preview] (Outlook) Add masterCategories

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by ElizabethSamuel-MSFT** [comment]
Please do not merge yet as we're await technical review. Thanks.

**Review by uniqueiniquity — CHANGES_REQUESTED** [review]
Marking as changes requested until technical review is completed.

File: types/office-js-preview/index.d.ts [hunk 1]
```diff
@@ -17555,6 +17571,143 @@ declare namespace Office {
...
+         * @remarks
+         * <table>
+         *   <tr><td>{@link https://docs.microsoft.com/outlook/add-ins/understanding-outlook-add-in-permissions | Minimum permission level}</td><td>ReadWriteMailbox</td></tr>
+         *   <tr><td>{@link https://docs.microsoft.com/outlook/add-ins/#extension-points | Applicable Outlook mode}</td><td>Compose or Read</td></tr>
+         *   <tr><td>Errors</td><td>PermissionDenied - The user does not have permission to perform this action.</td></tr>
+         * </table>
+         * 
+         * @beta
+         */
+        removeAsync(categories: CategoryDetails[], options?: Office.AsyncContextOptions, callback?: (asyncResult: Office.AsyncResult<void>) => void): void;
```
Comment (exextoc) [inline types/office-js-preview/index.d.ts]: This is not a CategoryDetails object array, it should be an array of category strings that correspond to the name of the categories in the master table.

**Review by exextoc — APPROVED** [review]

**Comment by ElizabethSamuel-MSFT** [comment]
We've received technical review. Thanks.

**Review by uniqueiniquity — APPROVED** [review]

**Pull request merged**